fractor/src/FileSystem/FileFinder.php

```diff
@@ -13,15 +13,18 @@ final class FileFinder
      */
     public function findFiles(array $directories, array $fileExtensions): array
```

Comment:
Should we maybe rename this to `findNonEmptyFiles`?

Reply:
No, i would leave the name as it is. Why we some change an empty file